sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_metrics_v1/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+friendly_name: Feature Usage Metrics
+description: |-
+  Metrics from metrics pings for mobile feature usage dashboards (Fenix)
```

Comment:
If there is any related DTMO documentation that could provide further context on these events then including a link here could be nice to make it easier in the future to find more detailed information.